rps.c

```diff
@@ -0,0 +1,57 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+/*Jack Rosen. The purpose is to play a game of rock, paper, scissors five times*/
```

Comment:
Good job, but you didn't implement all the instructions. You forgot to tell who won best of five games. Also, you should have used else if statements - it's more efficient to do so. Lastly, more commenting next time!